zuul.d/ansible-network-jobs.yaml

```diff
@@ -0,0 +1,10 @@
+---
+
+- job:
+    name: network-ee-integration-tests-resource_manager-ios
```

Comment:
this job should be calle

network-ee-integration-tests-ansible-network

so we reference the collection, not the target